nautobot_golden_config/nornir_plays/config_intended.py

```diff
@@ -42,7 +43,7 @@ def run_template(  # pylint: disable=too-many-arguments
         task (Task): Nornir task individual object
         logger (NornirLogger): Logger to log messages to.
         global_settings (GoldenConfigSetting): The settings for GoldenConfigPlugin.
-        nautobot_job (Result): The Nautobot Job instance being ran.
+        job_result (Result): The the output from the Nautobot Job instance being run.
```

Comment:
This was done on purpose, based on @jmcgill298 investigation, I do not think it should be reverted, see the following for more details https://github.com/nautobot/nautobot-plugin-golden-config/pull/158#discussion_r752840122